### ISSUE TITLE
Add Errors to proto-dc2_v5.0 galaxies

### ISSUE THREAD

**Issue by sschmidt23** [LSSTDESC/pz_pdf#3]
We need to add magnitude errors to the LSST ugrizy truth magnitudes for proto-dc2_v5.0 before we can run a small number of photo-z codes on them.  The goal is creating a "value added" catalog for the ~37 million proto-dc2_v5.0 galaxies that includes magnitude errors and true redshift as input to the photo-z codes.

**Comment by sschmidt23** [comment]
This is done in the issue/3/add_errs_todc2v5 branch, now need to do the same for CosmoDC2v1.1.4, creating a new issue for that.
